crates/libcontainer/src/container/tenant_builder.rs

```diff
@@ -148,13 +149,13 @@ impl TenantContainerBuilder {
         let mut notify_socket = NotifySocket::new(notify_path);
         notify_socket.notify_container_start()?;
 
-        close(write_end).map_err(LibcontainerError::OtherSyscall)?;
+        close(write_end.as_raw_fd()).map_err(LibcontainerError::OtherSyscall)?;
```

Comment:
Same question as the other comment, `write_end` as an `OwnedFd` will call close on drop ; so should we not call close here? If not, a better option to force the closing would be manually calling `drop()` on it at this point.

Reply:
Yes, since drop is automatically called at end of scope, the close() seems to be redundant.